salesagility/SuiteCRM
# Bug #164 Change AOS Product Images to use an entry point

Recreate way of refer the image in Products module
## Description


- modify and add values in modules/AOS_Products/vardevs.php
- in modules/AOS_Products/AOS_Products delete old way of save the image
- in modules/AOS_Products/views/view.edit.php delete old way of display the image
- also few changes in other files (see sourcecode)
  References issue #164
  
  
  ## Motivation and Context
  
  Product images currently links directly to the image in the upload folder. This only works if SuiteCRM is configured poorly and the upload folder is accessible (i.e. the .htaccess files are being ignored). We should change the product image code to refer to the image via the download entry point.
  ## How To Test This
  
- Create Products records,
- Attach images into Products records,
- Open DetailView to see Small copy of picture
- Open EditView, check if you can download image
  ## Types of changes
  
### Final checklist
